src/compile.c

```diff
@@ -750,31 +750,31 @@ typedef int (*yyrule)(yycontext *yy);\n\
 YY_PARSE(int) YYPARSEFROM(YY_CTX_PARAM_ yyrule yystart)\n\
 {\n\
   int yyok;\n\
-  if (!yyctx->__buflen)\n\
+  if (!yyctx->_buflen)\n\
     {\n\
-      yyctx->__buflen= YY_BUFFER_SIZE;\n\
-      yyctx->__buf= (char *)YY_MALLOC(yyctx, yyctx->__buflen);\n\
-      yyctx->__textlen= YY_BUFFER_SIZE;\n\
-      yyctx->__text= (char *)YY_MALLOC(yyctx, yyctx->__textlen);\n\
-      yyctx->__thunkslen= YY_STACK_SIZE;\n\
-      yyctx->__thunks= (yythunk *)YY_MALLOC(yyctx, sizeof(yythunk) * yyctx->__thunkslen);\n\
+      yyctx->_buflen= YY_BUFFER_SIZE;\n\
+      yyctx->_buf= (char *)YY_MALLOC(yyctx, yyctx->_buflen);\n\
+      yyctx->_textlen= YY_BUFFER_SIZE;\n\
+      yyctx->_text= (char *)YY_MALLOC(yyctx, yyctx->_textlen);\n\
+      yyctx->_thunkslen= YY_STACK_SIZE;\n\
+      yyctx->_thunks= (yythunk *)YY_MALLOC(yyctx, sizeof(yythunk) * yyctx->_thunkslen);\n\
 #ifdef __cplusplus\n\
-      yyctx->__vals = new std::vector<YYSTYPE>();\n\
-      yyctx->__vals->reserve(YY_STACK_SIZE);\n\
+      yyctx->_vals = new std::vector<YYSTYPE>();\n\
+      yyctx->_vals->reserve(YY_STACK_SIZE);\n\
 #else\n\
-`     yyctx->__valslen= YY_STACK_SIZE;\n\
```

Comment:
How weird -- Any idea why there was a back-quote character at the start of this line?

Reply:
I have no clue, but it caused a compile error, and I couldn't see one anywhere else, so I guess it was a typo